learn/getting_started/quick_start.md

```diff
@@ -62,6 +62,8 @@ docker run -it --rm \
 
 Data written to a **Docker container is not persistent** and is wiped every time the container is stopped. We recommend using a shared Docker volume between containers and host machines to provide persistent storage.
 
+This method is not recommended for macOS and Windows, because I/O between the host and the container terribly slow.
```

Comment:
```suggestion
This method is not recommended for macOS or Windows, because I/O between the host and the container can be very slow. For these operating systems, we recommend using the native application or a [cloud-hosted option](#cloud-deploy).
```